docs/configuring-playbook-dimension.md

```diff
@@ -0,0 +1,48 @@
+# Setting up Dimension (optional)
+
+Dimension integrations manager installation is disabled by default. You can enable it in your configuration file (`inventory/matrix.<your-domain>/vars.yml`):
+
```

Comment:
An important prerequisite for Dimension working is the `dimension.<your-domain>` DNS record being set up correctly.

Maybe it makes sense to mention it again here (and link to `configuring-dns.md`), for those that have ignored setting up the record until now.